README.md

```diff
@@ -1,55 +1,93 @@
 # mSupply Mobile
 
-A user friendly, offline-first mobile app for use with [mSupply](http://msupply.org.nz). Provides electronic inventory control for medical stock. Designed and built by [Sustainable Solutions](http://sussol.net) for use in developing countries, the app is now open source and free to use. Hello
+Cross-platform, offline-first mobile application designed and developed by [Sustainable Solutions](http://sussol.net). Integrates with [mSupply](https://www.msupply.org.nz/) to provide accessible and user-friendly electronic inventory control for medical stock. Open-source and free-to-use, mSupply Mobile is designed and built with a specific focus on the needs of developing countries.
 
-### Features
+## Features
 
-- Receiving and issuing goods with automatic inventory adjustments
-- Stocktaking
-- Forecasts of required stock based on usage
-- Ordering stock from suppliers
-- Viewing stock on hand
-- Syncs with an mSupply server when internet is available
+- Track stock movements and view stock on hand.
+- Receive and issue goods with automatic inventory adjustments.
+- Order stock from suppliers.
+- Forecast required stock based on usage.
+- Sync data with a local or remote mSupply server when internet is available.
 
-See http://msupply.org.nz/mobile for a detailed description of these and other features
+See [http://msupply.org.nz/mobile](http://msupply.org.nz/mobile) for more details of these and many other features.
 
-### Building from Source
+## Getting started
 
-See the instructions on the [setup page](https://github.com/sussol/mobile/wiki/Setup#building-from-source)
+### Prerequisites
 
-### Use with an mSupply Server
+#### Android SDK
 
-Instructions on the [setup page](https://github.com/sussol/mobile/wiki/Setup#setting-up-msupply-server)
+- Install Android Studio and SDK tools: https://developer.android.com/studio.
+- Install SDKMAN for managing Java versions: https://sdkman.io/.
 
-### Set Up for Development
+#### React Native
 
-Instructions on the [setup page](https://github.com/sussol/mobile/wiki/Setup)
+- Install nvm for managing Node versions: https://github.com/nvm-sh/nvm.
```

Comment:
As a suggestion, a new line can be added after this one, mentioning which is the node.js version that need to be install/selected after having nvm running properly (e.g. node v10.13.0). That can avoid future problems during the installation.

Reply:
We have a `.nvm` file in the repo, so `nvm use .` should take care of that :).